scripts/setup_linux.sh

```diff
@@ -102,6 +102,15 @@ print_warning() {
   printf "\e[0;33m%s\e[0m" "${1}"
 }
 
+# Verify this script is running from the correct folder (root directory)
+dir_err_txt="Please run this script only from the root of the garage \
+repository, i.e. you should run it using the command \
+\"bash scripts/setup_linux.sh\""
+if ! test -f setup.py && ! grep -Fq "name='rlgarage'," setup.py; then
```

Comment:
@gonzaiva don't you use *either* `test -f` *or* `if`

Reply:
Ok, I will only use one from now on

Reply:
haha. i was asking you what's appropriate according to convention/style guide.

Reply:
Just replaced the test statement with brackets